packages/ts-config/tsconfig.json

```diff
@@ -6,7 +6,6 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "incremental": true,
-    "allowSyntheticDefaultImports": true,
-    "noImplicitAny": true
```

Comment:
this is redundant with `"strict": true` (in base tsconfig) - shouldn't have added it, but since it doesn't change anything I'm not gonna publish a new package